File: tests/conformance/fixtures/python/27-agent-mandate.py
from sykli import Pipeline, actor, exit_code, file_evidence_non_empty, mandate

p = Pipeline()
p.task("implement").run("go test ./...").task_type("test").success_criteria([
    exit_code(0),
]).evidence_required([
    file_evidence_non_empty("coverage", "coverage.out"),
]).actor(actor("agent", "claude")).mandate(
    mandate(["sdk/**", "tests/conformance/**"])
)
p.emit()

File: tests/conformance/fixtures/elixir/27-agent-mandate.exs
use Sykli

pipeline do
  task "implement" do
    run("go test ./...")
    task_type(:test)
    success_criteria([exit_code(0)])
    evidence_required([file_evidence_non_empty("coverage", "coverage.out")])
    actor(:agent, "claude")
    mandate(["sdk/**", "tests/conformance/**"])
  end
end
